app/controllers/your-psp/get.controller.test.js

```diff
@@ -36,10 +38,21 @@ describe('Your psp GET controller', () => {
 
 
   it('should pass the enableStripeOnboardingTaskList to the page ', async () => {
```

Comment:
Change this to something like:
`when enableStripeOnboardingTaskList=true, it should pass that to the page`